packages/ember-toucan-core/CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # @crowdstrike/ember-toucan-core
 
+## 0.4.0
+
+### Minor Changes
+
+- [#254](https://github.com/CrowdStrike/ember-toucan-core/pull/254) [`532ab89`](https://github.com/CrowdStrike/ember-toucan-core/commit/532ab8997917c9dc4beb3da93e19578ec73b3f09) Thanks [@clintcs](https://github.com/clintcs)! - Add Multiselect test helpers.
```

Comment:
I really haven't read the whole semantic spec so...but if you have 3 minor changes and 2 patch, wouldn't it be like

0.3.1
+ 3
+.   2
-----
0.5.3 ?

Reply:
Changesets batches changes in a release and takes the "largest" one.  So if you have:

- minor
- patch

It will only update the package to `minor`.  If we would've cut releases as we merged PRs, we would have gotten what you described above, but since we allowed the changes to batch, we get the "highest" breaking change as the new version.

Reply:
I think the minor changes `reset` the patch change counter to 0, and all the minor changes are happening at the same time, which is like one big minor change, so 0.4.0 makes sense to me

Reply:
Or when you update `minor` does it zero out the patch version?

Reply:
TIL thanks folks!